crates/matrix-sdk-common/src/linked_chunk/mod.rs

```diff
@@ -945,7 +946,7 @@ impl PartialEq<u64> for ChunkIdentifier {
 ///
 /// It's a pair of a chunk position and an item index.
 #[derive(Copy, Clone, Debug, PartialEq)]
-pub struct Position(ChunkIdentifier, usize);
+pub struct Position(pub(super) ChunkIdentifier, pub(super) usize);
```

Comment:
nit: no need to make thse public, they're already public via `Position::chunk_identifier()` and `Position::index()`.

Reply:
It's useful to be public to build them. Gonna check if it's still really required. I wouldn't want to introduce a constructor just for such simple types.

Reply:
Okay, let's use constructor.